.changeset/silly-bees-beam.md

```diff
@@ -2,4 +2,6 @@
 'openzeppelin-solidity': major
 ---
 
-`ERC20Votes`: Changed internal vote accounting to reusable `Votes` module previously used by `ERC721Votes`. Removed implicit `ERC20Permit` inheritance. [#3816](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3816)
+`ERC20Votes`: Changed internal vote accounting to reusable `Votes` module previously used by `ERC721Votes`. Removed implicit `ERC20Permit` inheritance. Note that the `DOMAIN_SEPARATOR` getter was previously guaranteed to be available for `ERC20Votes` contracts, but is no longer available unless `ERC20Permit` is explicitly used; ERC-5267 is recommended as an alternative.
```

Comment:
Should we b more explicit that ERC-5267 is included by default (in `EIP721.sol`)?

Reply:
Added a note about that.